### PR TITLE
Add table models

### LINKED ISSUE #112: Table_Functions

### PR DESCRIPTION
**Pull request by mattiasbarth**

### CODE REVIEW

File: app/Data/models/comment_like.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from app.Data.db import Base
+import sqlalchemy as sa
+
+
+class CommentLike(Base):
```
Comment (AntonMaxen) [inline app/Data/models/comment_like.py:5]: No syntax errors, only structure that differs from others

**Review by HenrikLiman — APPROVED** [review]

File: app/Data/models/picture_like.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from app.Data.db import Base
+import sqlalchemy as sa
+
+
+class PictureLike(Base):
```
Comment (AntonMaxen) [inline app/Data/models/picture_like.py:5]: looks good

File: app/Data/models/review.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import sqlalchemy as sa
+
+
+class Review(Base):
+    __tablename__ = 'review'
+
+    Id = sa.Column(sa.Integer, nullable=False, autoincrement=True)
+    Title = sa.Column(sa.String(45), nullable=False)
+    ReviewText = sa.Column(sa.String(1024), nullable=False)
+    Score = sa.Column(sa.DECIMAL(2, 1), nullable=False)
```
Comment (AntonMaxen) [inline app/Data/models/review.py:11]: looks good

File: app/Data/models/comment_like.py (hunk 1 above)
Comment (AntonMaxen) [reply app/Data/models/comment_like.py:5]: Written with linewidth 79 in mind that is defined by PEP 8, all collaborators will change to these style rules.

**Review by AntonMaxen — APPROVED** [review]

**Pull request merged**